commands/example_commands/deploy

```diff
@@ -75,19 +76,94 @@ display_deployment_details(){
 }
 
 
+deploy_from_github(){
+  # Check and remove existing grpl-gras-example folder
+  if [ -d "/tmp/grpl-gras-examples" ]; then
+    if ! rm -rf /tmp/grpl-gras-examples; then
+      status_log $TYPE_ERROR "Failed to remove existing grpl-gras-examples directory"
+      exit 1
+    fi
+  fi
+
+  # Clone the repository
+  if ! eval "$(getGumSpinnerOrLogger "cloning grpl-gras-examples repository") git clone https://github.com/zahid-iqbal-alpha/grpl-gras-examples.git /tmp/grpl-gras-examples"; then
```

Comment:
please create a repo in grapple-solution github org for that...
I think you should have the permissions to do so...
https://github.com/orgs/grapple-solution/repositories

Reply:
👍